docs/UserGuide.md

```diff
@@ -426,6 +430,115 @@ Invalid Input Example | Application Output
 
 
 ## Payments
+
+CampusConnect allows you to keep track of payments between you and your friends! Each contact has a **balance**, which is the amount of money that you owe them (or that they owe you). This balance is positive if they owe you money, and negative if you owe them money.
```

Comment:
not sure if i might be nitpicking too much but should we change `friends` to `contacts` for standardization of persona?

Reply:
Yes! Good point